src/pages/landing/ebooks/components/_get-ebook.tsx

```diff
@@ -343,9 +347,7 @@ const GetEbook = ({ color = 'var(--color-white)', ebook_utm_code, onSubmit }: Ge
                             components={[
                                 <StyledLocalizedLink
                                     key={0}
-                                    to="/tnc/security-and-privacy.pdf"
-                                    size="1.2rem"
-                                    color="red"
+                                    href="/tnc/security-and-privacy.pdf"
```

Comment:
```suggestion
                            url={{
                                target: '_blank',
                                type: 'non-company',
                                href: `/tnc${is_eu ? '/eu' : ''}/security-and-privacy.pdf`,
                            }}
 ```

Reply:
Since the security and privacy PDF is different for `eu` and `non-eu` let's have different links.
May you also add `useRegion()` on top of your component?